src/access/Privilege.h

```diff
@@ -38,5 +38,24 @@ enum class Privilege : uint8_t
     kAdminister = 1 << 4
 };
 
+// Function used to validate that the parameter 'mask' contains all the values
+// defined inside the enum class "Privilege", and only those values.
+constexpr auto kPrivilegeMaskValidation(uint8_t mask)
```

Comment:
```suggestion
constexpr bool kPrivilegeMaskValidation(uint8_t mask)
```

Reply:
can we also use underlying_type_t for the mask type?

Reply:
Change name to not be `k.....` those are constant names. It should be something like `IsValidPrivilegeMask` or similar.